scenarios3/14_Chasing_Dragons.cfg

```diff
@@ -325,19 +326,20 @@
             hitpoints=61
             x,y=10,13
         [/unit]
+#endif
     [/side]
     [side]
         type=Orcish Warlord
         id=warlord
         generate_name=yes
         random_traits=yes
         side=4
-        canrecruit=yes
+        canrecrut=yes
```

Comment:
Typo?

Reply:
yes, I fixed a bunch of those the other day, but I must have been working from an old copy.  Or I just missed one completely.